### PR TITLE
docs: add stable-horde type

### PR DESCRIPTION
**Pull request by MaikoTan**

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -4,10 +4,10 @@
 
 ### type
 
-- 类型：`'login' | 'token' | 'naifu' | 'sd-webui'`
+- 类型：`'login' | 'token' | 'naifu' | 'sd-webui' | 'stable-horde'`
 - 默认值：`'token'`
 
-登录方式。`login` 表示使用账号密码登录，`token` 表示使用授权令牌登录。`naifu` 和 `sd-webui` 对应着其他类型的后端。
+登录方式。`login` 表示使用账号密码登录，`token` 表示使用授权令牌登录。`naifu`、 `sd-webui` 和 `stable-horde` 对应着其他类型的后端。
```
Comment (shigma) [inline docs/config.md:10]:
```suggestion
登录方式。`login` 表示使用账号密码登录，`token` 表示使用授权令牌登录。`naifu`、`sd-webui` 和 `stable-horde` 对应着其他类型的后端。
```

pangu

File: docs/index.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 - 自定义违禁词表
 - 中文关键词自动翻译
 - 发送一段时间后自动撤回
-- 连接到私服 · [SD-WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
+- 连接到私服 · [SD-WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui) · [Stable Horde](https://stablehorde.net/)
```
[the remark this answers is not in the document]
Comment (shigma) [reply docs/index.md:11]: and package.json 🤣

File: docs/config.md (hunk 1 above)
Comment (MaikoTan) [reply docs/config.md:10]: Oh, so the dunhao symbol is not panguable.

File: docs/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MaikoTan) [reply docs/index.md:11]: Well, `and else` in `package.json` covered it already I think.

[the remark this answers is not in the document]
Comment (shigma) [reply docs/index.md:11]: readme does not have an `and else`.

**Review by shigma — APPROVED** [review]
善

**Pull request merged**